Apps/frontend/src/app.scss

```diff
@@ -336,6 +340,10 @@ nav {
   }
 }
 
+.icon {
```

Comment:
Would prefer if we merged your `icon` and `icon-button`. Maybe use something like `.icon { &--button { ... } &--tooltip { ... } /*Icon definitions*/ &--on { ... ] &--off { ... } ... }`

Reply:
I think the .icon-button do not need some of the properties that .icon has. Maybe we could make a selector to encapsulate the background-position and background-repeat but I think this wouldn't make the code easier to read. What do you think?